go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.29.0
-	github.com/rancher/rke v1.4.11
+	github.com/rancher/rke v1.4.15
```

Comment:
can you bump rke to 1.4.16 which is the latest?

Reply:
Can you also update the PR's title and description to use RKE 1.4.16?

Reply:
Done, I updated it in the PR title and in the code